packages/devtools_test/lib/src/mocks.dart

```diff
@@ -241,6 +247,15 @@ class FakeServiceManager extends Fake implements ServiceConnectionManager {
 
   @override
   ValueListenable<bool> get deviceBusy => ValueNotifier(false);
+
+  @override
+  Future<void> vmServiceOpened(
+    VmServiceWrapper service, {
+    Future<void> onClosed,
+  }) {
+    resolvedUriManager.vmServiceOpened();
+    return Future.value();
```

Comment:
you shouldn't have to return here

Reply:
linter doesn't seem to like when I remove this.